### DISCUSSION TITLE
ModuleNotFoundError: No module named 'numpy'

### DISCUSSION THREAD

**Discussion by kkuru1** [carla-simulator/carla#5320] [General]
Hi
I installed CARLA 0.9.13 Release for Windows.
When I run the examples as follows, it gives an error about numpy even though it is installed. 

![image_2022-04-03_154733](https://user-images.githubusercontent.com/45274921/161433633-3ba6dc77-fff8-4a16-a29b-d586dc9bb881.png)

![image_2022-04-03_154544](https://user-images.githubusercontent.com/45274921/161433557-32fcfed1-ec3b-4ea2-a3a7-f7770a91bee8.png)

Thanks in advance.

KK

**Comment by iammix** [comment]
Your python does not have numpy. 
Execute "py -37 -m pip install numpy" in order to install it.

**Reply by kkuru1** [reply]
It has worked perfectly, Many Thanks Iammix, highly appreciated.

**Reply by DessyMaramis** [reply]
@iammix can you explain how to install numpy on linux? 
Thank you
